python/paddle/audio/features/layers.py

```diff
@@ -31,6 +31,8 @@
     'hamming',
     'hann',
     'kaiser',
+    'bartlett',
+    'nuttall',
```

Comment:
because of adding types of window functions, we should synchronously expand the introduction documents of `window` member in `Spectrogram`, `MelSpectrogram`, `LogMelSpectrogram`, and `MFCC` classes below in this file.

Reply:
好的好的